pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -8,14 +8,15 @@ import (
 	"time"
 
 	log "github.com/sirupsen/logrus"
+	coordv1beta1 "k8s.io/api/coordination/v1beta1"
```

Comment:
Make sure that k8s version and the rest of the packages in https://github.com/kubevirt/node-maintenance-operator/blob/master/Gopkg.toml are aligned with the version we need here.